scripts/getVersions.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+/**
+ * Read versions.json file from the appropriate branch in platform
+ * Example
+ *   ./scripts/getVersions.js
+ */
+
+// Increase this if platform major changes
+const master = '18.0'
+
+const https = require("https");
+const xml2js = require('xml2js');
+const fs = require('fs');
+
+async function main() {
+  const parentJs = await xml2js.parseStringPromise(fs.readFileSync('pom.xml', 'utf8'));
+  const version = parentJs.project.version[0];  
+  const branch = version.replace(/^(\d+\.\d+).*$/, '$1');
+
```

Comment:
I  suggest we test if the branch exists by making a HEAD request to `https://api.github.com/repos/vaadin/platform/branches/${version)`. A 404 indicates that it does not exist, so we should use master. That way we don't need to hard code the master value and remember to update it manually.

Reply:
Done